maven-java/kalix-maven-plugin/src/main/java/kalix/DeployMojo.java

```diff
@@ -36,30 +35,85 @@ public class DeployMojo extends AbstractMojo {
     @Parameter(property = "kalixContext")
     private String kalixContext;
 
+    @Parameter(property = "kalixCurrentProject", required = true)
+    private String kalixCurrentProject;
+
+    @Parameter(defaultValue = "false", property = "kalixDeploymentOn", required = true)
```

Comment:
I would call it something like: `kalixDeploymentEnabled` or maybe `deployToKalix`. 

In the pom, we will see something like: `<deployToKalix>true</deployToKalix>`

Reply:
Yeah, I like `deployToKalix`